extra_foam/gui/image_tool/reference_view.py

```diff
@@ -65,6 +77,21 @@ def updateF(self, data, auto_update):
             # is deferred.
             self._reference.setImage(data.image.reference)
 
+        if self._is_recording:
+            # Update cached image and count
+            image = self._corrected.image.astype(np.float32)
+            if self._recorded_image is None:
+                self._recorded_image = image.astype(np.float32)
+            else:
+                # Average without storing images
+                num_images = self._count + 1
```

Comment:
`num_images` is unnecessary if you increment `self._count` at the beginning of the if-statement.

Reply:
I was thinking of only adding the count after the operations are finished, but I guess it's unlikely that the calculations would result into error XD will change.